Makefile

```diff
@@ -210,32 +176,14 @@ compdb:
 tidy:
 	$(RUN) tidy
 
-.PHONY: clear_xcode_cache
-clear_xcode_cache:
-ifeq ($(BUILD), osx)
```

Comment:
@jfirebaugh @1ec5 is it really ok that we will no longer clear the derived data folder for osx builds?